soc/atmel/sam/sam4l/Kconfig.soc

```diff
@@ -4,12 +4,6 @@
 config SOC_SERIES_SAM4L
 	bool
 	select SOC_FAMILY_ATMEL_SAM
-	help
-	  Enable support for Atmel SAM4L Cortex-M4 microcontrollers.
-	  Part No.: SAM4LS8C, SAM4LS8B, SAM4LS8A, SAM4LS4C, SAM4LS4B,
-	  SAM4LS4A, SAM4LS2C, SAM4LS2B, SAM4LS2A, SAM4LC8C, SAM4LC8B,
-	  SAM4LC8A, SAM4LC4C, SAM4LC4B, SAM4LC4A SAM4LC2C, SAM4LC2B,
-	  SAM4LC2A
```

Comment:
could be very useful when grepping for part number (e.g. what `SOC_SERIES` do I need to select for `SAM4LC2C`?)

Reply:
You don't select a SOC_SERIES in hwmv2, it's done for you by selecting the SOC